reports/report_builder.py

```diff
@@ -3403,8 +3403,7 @@ def ws_107(self, ws):
                 for row in rows:
                     counts.update({(row[sq_field], TOPIC_GROUPS[row['topic']]): row['n']})
 
-                secondary_counts[media_type] = counts
-                
+                secondary_counts[media_type] = Counter(counts)
```

Comment:
Why not leave
```suggestion
                secondary_counts[media_type] = counts
```
and just move the `counts` [initialization](https://github.com/CodeForAfrica/gmmp/pull/171/files#diff-19a686dbe88bb9f2f8c4f2ce5e331aa6f6db260926a2a5b7f1edfa7864a649b4L3392-L3393) inside the `for` loop:
```python
....
            for media_type, model in person_models.items():
                counts = Counter()
                sheet_name = model.sheet_name()
...
```
That way it's clear we need new `counts` for each media.
```

Reply:
yeah. This makes more sense.